backend/vagrant.pm

```diff
@@ -0,0 +1,186 @@
+# Copyright © 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package backend::vagrant;
+
+use strict;
+use warnings;
+
+use base 'backend::virt';
+use bmwqemu;
+use testapi ();
+
+use Mojo::Base -signatures;
```

Comment:
Looks like you've enabled signatures but don't make use of them. It would be nice to make use of them.

Reply:
Oh, I thought this would do $magic by itself, sorry… I'll try to fix that.

Reply:
Like this?

Reply:
Yes, looks good now :-)